.gitignore

```diff
@@ -52,3 +52,5 @@ build-iPhoneSimulator/
 # unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
 .rvmrc
 
+# ctags
+tags
```

Comment:
It's great to have `ctags` incase somebody uses them for development, in my case i use it in combination with vim + autocomplete to jump between methods efficiently and autocomplete